content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -378,6 +379,254 @@ PS > Get-Service kubelet; Get-Service kube-proxy;
 CMD > sc.exe queryex kubelet && sc qc kubelet && sc.exe queryex kube-proxy && sc.exe qc kube-proxy
 ```
 
+### Group Managed Service Accounts (GMSA) support
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+Starting with Kubernetes v1.14, Windows container workloads can be configured to use [Group Managed Service Accounts](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA). Group Managed Service Accounts are a specific type of Active Directory account that provides automatic password management, simplified service principal name (SPN) management, and the ability to delegate the management to other administrators across multiple servers.
+
+In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.14, the only container runtime interface that supports GMSA for Windows workloads is Dockershim. Implementation of GMSA through CRI and other runtimes is planned for the future.
+
```

Comment:
NOTE: Currently this feature is in alpha support.  While the overall goals and functionality will not change, the way in which the GMSA information is passed to the container runtime may.  Please take this into consideration when testing or adopting this feature.

Reply:
We already have the tag `{{< feature-state for_k8s_version="v1.14" state="alpha" >}}` but I guess no harm in re-iterating alpha support.